### PR TITLE
Add build script for snappy_java

### PR DESCRIPTION
**Pull request by krishvoor**
Add's build script for snappy_java

Signed-off-by: Krishna Harsha Voora <krishvoor@in.ibm.com>

### CODE REVIEW

File: s/snappy_java/build.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# ----------------------------------------------------------------------------
+# Package       : lz4-java
```
Comment (Abhijit-Mane) [inline s/snappy_java/build.sh:2]: minor typo, should be: "**snappy-java**"

File: s/snappy_java/build.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+export PATH=${PATH}:/usr/local/bin/apache-maven-3.6.3/bin/
+
+# 
+
+# copy build artifacts
+cp -f ./target/snappy-java*.jar ../
+
+# cleanup
+cd ..
+rm -rf lz4-java
```
Comment (Abhijit-Mane) [inline s/snappy_java/build.sh:46]: minor typo, should be: "**rm -rf snappy-java**"

File: s/snappy_java/build.sh (hunk 1 above)
Comment (Abhijit-Mane) [inline s/snappy_java/build.sh:2]: minor typo, should be: "**snappy-java**"

File: s/snappy_java/build.sh (hunk 2 above)
Comment (Abhijit-Mane) [inline s/snappy_java/build.sh:46]: minor typo, should be: "**rm -rf snappy-java**"

File: s/snappy_java/build.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+if [ "$BRANCH" == "" ]
+then
+	echo "BRANCH = master"
+else
+	echo "BRANCH = $BRANCH"
+fi
+
+git clone $BRANCH https://github.com/xerial/snappy-java || (echo "git clone failed"; exit $?)
+cd snappy-java
+
```
Comment (Abhijit-Mane) [inline s/snappy_java/build.sh:34]: Missed to run "**make**" to trigger snappy-java build ?

**Review by Abhijit-Mane — APPROVED** [review]
minor comments, otherwise lgtm.

File: s/snappy_java/build.sh (hunk 1 above)
Comment (krishvoor) [reply s/snappy_java/build.sh]: You know where I got the inspiration from ;-)

So, I have updated & pushed a PR, please help review.

File: s/snappy_java/build.sh (hunk 3 above)
Comment (krishvoor) [reply s/snappy_java/build.sh:34]: Updated, thank you!

**Review by Abhijit-Mane — APPROVED** [review]

**Review by Abhijit-Mane — APPROVED** [review]
lgtm

**Pull request merged**